crates/control/src/role_assignment.rs

```diff
@@ -86,19 +88,10 @@ impl RoleAssignment {
         Ok(Self {
             last_received_spl_striker_message: None,
             last_system_time_transmitted_game_controller_return_message: None,
-            last_transmitted_spl_striker_message: None,
+            last_transmitted_striker_message: None,
             role: Role::Striker,
             role_initialized: false,
```

Comment:
try to remove this variable. this is only used in the very first cycle to initialize `self.role`

Reply:
Removed